com.github.wjaguar.mtpaint.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: com.github.wjaguar.mtpaint
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+command: mtpaint
+rename-icon: mtpaint
+rename-desktop-file: mtpaint.desktop
+rename-appdata-file: mtpaint.appdata.xml
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Network access
+  # for printing
+  - --share=network
+  # Filesystem access
+  # xdg-desktop-portal-gtk does not support Gtk+2
+  # even if, this program requires direct access to files
+  - --filesystem=host
```

Comment:
Surely not to entire host.

Reply:
> Surely not to entire host.

Almost every graphics editor here has the same privileges level:
https://github.com/flathub/org.gimp.GIMP/blob/b43dc36fbcbc1034b54f701d96616d5954e60277/org.gimp.GIMP.json#L13
https://github.com/flathub/org.kde.krita/blob/9c41b5c25dfe589bb3b4b54fc814889e2d3091f6/org.kde.krita.yaml#L14
https://github.com/flathub/org.inkscape.Inkscape/blob/4d9be1cd3d918bad54463b800494ec2c415ec2b5/org.inkscape.Inkscape.json#L13
https://github.com/flathub/org.pencil2d.Pencil2D/blob/a910c55c823d3e4b67be68dffc9bf8532dcbfd6d/org.pencil2d.Pencil2D.yaml#L15
https://github.com/flathub/org.darktable.Darktable/blob/2c64ac6da84cc8068006809a46e0340b65740333/org.darktable.Darktable.json#L14

Reply:
As I said previously, rules aren't set in stone and not everything would be accepted as is today. No amount of throwing links at me will change it.

Reply:
> As I said previously, rules aren't set in stone and not everything would be accepted as is today. No amount of throwing links at me will change it.

If you prefer to create useless tools, go ahead, do it, but without me.

Reply:
Less than two weeks ago you gave the green light for fontforge, which requires the same filesystem permissions. But of course, `rules are fluid`...
https://github.com/flathub/flathub/pull/724#issuecomment-440909514
https://github.com/flathub/io.github.Fontforge/blob/7d3d0fcf461beb5cdeaacaedc97dff0241361eb7/io.github.Fontforge.yaml#L16

Reply:
No reason to get so passively aggressive. I didn't raise it about Fontforge because it's not obvious what it should be limited to by default, as opposed to painting program.

Reply:
There is no difference here between a graphic editor and a font editor. Both should have the same level of filesystem access.